src/datadog/tracer_config.h

```diff
@@ -13,6 +13,7 @@
 #include "datadog_agent_config.h"
 #include "error.h"
 #include "expected.h"
+#include "json.hpp"
```

Comment:
Is `nlohmann::json` used in this header? If not, remove this include. If so, include `json_fwd.hpp` instead.